video/image_writer.c

```diff
@@ -430,11 +409,8 @@ static bool write_avif(struct image_writer_ctx *ctx, mp_image_t *image,
         goto free_data;
     }
 
-    ret = avio_open(&avioctx, filename, AVIO_FLAG_WRITE);
-    if (ret < 0) {
-        MP_ERR(ctx, "Could not open file '%s' for saving images\n", filename);
-        goto free_data;
-    }
+    avio_open_dyn_buf(&avioctx);
+    MP_HANDLE_OOM(avioctx);
```

Comment:
I guess it is fine, but I'm not a big fun of using OOM macro for functions that are not pure "allocate". It is external lib implementation detail that only possible error is `AVERROR(ENOMEM)`, but I don't think it is clear from API pov.

Either way, no need to change that, just wanted to say it.